src/App.jsx

```diff
@@ -9,32 +10,25 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
```

Comment:
it's better to use the full tab here
```suggestion
  const [selectedTab, setSelectedTab] = useState(tabs[0]);
```